packages/sdk/src/commons.ts

```diff
@@ -13,36 +13,34 @@ import {
 } from "viem";
 import { sepolia, arbitrumSepolia } from "viem/chains";
 import { ChainId, DEPLOYMENT_ADDRESSES } from "@carrot-kpi/contracts";
+import { Environment } from "@carrot-kpi/shared-state";
 
 // reexport chain id
 export { ChainId } from "@carrot-kpi/contracts";
 
 export const CACHER = new Cacher("carrot-kpi-sdk");
 
-export enum Environment {
-    Development = "development",
-    Staging = "staging",
-}
-
 export interface ServiceUrls {
-    staticCdn: string;
-    dataCdn: string;
     dataManager: string;
 }
 
 export const SERVICE_URLS: Record<Environment, ServiceUrls> = {
+    [Environment.Local]: {
+        dataManager: "https://data-manager.api.dev.carrot.community",
```

Comment:
You are right. We should probably have a local S3 deployment in `carrot-scripts` alongside the other services (it would replace the current IPFS node start up). Maybe I should add a todo and do this later?

Reply:
Yes, let's add an issue/todo for this, and we can do it later.

Reply:
Issue added: https://github.com/carrot-kpi/monorepo/issues/619